dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -533,6 +533,42 @@ public Long execute(final Jedis client, final ConnectionContext state) throws Dy
         });
     }
 
+    
+    @Override
+    public ScanResult<Map.Entry<String, String>> hscan(final String key, final int cursor) {
+    	return d_hscan(key, cursor).getResult();
+    }
+    
+    public OperationResult<ScanResult<Map.Entry<String, String>>> d_hscan(final String key, final int cursor){
+    	
+    	return connPool.executeWithFailover(new BaseKeyOperation<ScanResult<Map.Entry<String, String>>>(key, OpName.HSCAN) {
```

Comment:
You're correct - hscan is the only operation that needs it (not sscan or zscan). Thanks!